### PR TITLE
Porting packages in master to ROS 2

### PR DESCRIPTION
**Pull request by ahcorde**
List of packages to port:

 - [x] proposed_aerial_msgs
 - [x] ir_beacon
 - [x] px4
 - [x] qgroundcontrol
 - [x] sitl_launcher

### CODE REVIEW

**Comment by tfoote** [comment]
We'll need this. Though we may need to keep two copies or make it dual compatible for side by side operations. But lets go ahead with this for now.

**Comment by ahcorde** [comment]
what should I do with the package ir_beacon? port to ROS 2? is used for something?

File: sitl_launcher/launch/demo.launch.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+def generate_launch_description():
+
+    declare_sitl_world_cmd = DeclareLaunchArgument(
+        'sitl_world',
+        default_value='yosemite',
+        description='World [yosemite, mcmillan, ksql, baylands]')
+
+    sitl_world = ReplaceWorldString(sitl_world=launch.substitutions.LaunchConfiguration('sitl_world'))
+
+    included_launch = launch.actions.IncludeLaunchDescription(
```
Comment (tfoote) [inline sitl_launcher/launch/demo.launch.py]: I"m not sure were it should be but, the `sitl_world` argument should have a default. Right now it crashes out if unset on the command line.

```
Executing command: 
docker run -it   --rm    -v /home/tfoote:/home/tfoote   --gpus all -v /run/user/1000/pulse:/run/user/1000/pulse --device /dev/snd  -e PULSE_SERVER=unix:/run/user/1000/pulse/native -v /run/user/1000/pulse/native:/run/user/1000/pulse/native --group-add 29   -e DISPLAY -e TERM   -e QT_X11_NO_MITSHM=1   -e XAUTHORITY=/tmp/.docker.xauth -v /tmp/.docker.xauth:/tmp/.docker.xauth   -v /tmp/.X11-unix:/tmp/.X11-unix   -v /etc/localtime:/etc/localtime:ro    038b320bf156 
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-tfoote'
[INFO] [launch]: All log files can be found below /home/tfoote/.ros/log/2019-11-13-13-43-52-395839-c47a23c3448a-10
[INFO] [launch]: Default logging verbosity is set to INFO
[ERROR] [launch]: Caught exception in launch (see debug for traceback): launch configuration 'sitl_world' does not exist
[INFO] [rviz2]: Stereo is NOT SUPPORTED
[INFO] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[INFO] [rviz2]: Stereo is NOT SUPPORTED
```

File: sitl_launcher/launch/demo.launch.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        launch.launch_description_sources.PythonLaunchDescriptionSource(
+                sitl_world))
+
+    use_rviz = LaunchConfiguration('use_rviz')
+    declare_use_rviz_cmd = DeclareLaunchArgument(
+        'use_rviz',
```
Comment (tfoote) [inline sitl_launcher/launch/demo.launch.py]: When I launch this with True I get two copies of rviz, one with the config and one plain. 

And when it's false I just get the plain rviz. 

Screenshot from 2019-11-13 15-25-53

File: sitl_launcher/launch/demo.launch.py (hunk 1 above)
Comment (ahcorde) [reply sitl_launcher/launch/demo.launch.py]: This commit solve the issue https://github.com/ahcorde/drone_demo/commit/a11b62ca95aaf6fda5963eb209f3a401ef0388f8

File: sitl_launcher/launch/demo.launch.py (hunk 2 above)
Comment (ahcorde) [reply sitl_launcher/launch/demo.launch.py]: @tfoote can you verify your entrypoint.sh ?

Comment (ahcorde) [reply sitl_launcher/launch/demo.launch.py]: The one with the config file up it's the rviz launched by the launch file but I included something in the entrypoint to launch an rviz instance but I removed it https://github.com/osrf/drone_demo/commit/373fb54242631849e78867ae12e2db3ad18298e5#diff-b958f585a04af5ee2087610ea7180f34

File: drone_node/include/utils/geodetic_converter.hpp [hunk 3]
```diff
@@ -0,0 +1,71 @@
+#ifndef RVIZ_AERIAL_PLUGINS_GEODETIC_CONVERTER_HPP_
```
Comment (tfoote) [inline drone_node/include/utils/geodetic_converter.hpp]: This include guard is wrong. Did the code get copied over? Should we be refactoring this into somewhere more convenient?

Comment (ahcorde) [reply drone_node/include/utils/geodetic_converter.hpp]: The code is originally from this repo. I included in the header https://github.com/osrf/drone_demo/blob/1e7837279fd0af83201f259564b22f91d45f4a39/drone_node/include/utils/geodetic_converter.hpp

How should I include this?

**Pull request merged**